src/plugins/intel_cpu/docs/performance_hint_and_threads_scheduling.md

```diff
@@ -0,0 +1,97 @@
+# Performance Hint and Threads Scheduling 
```

Comment:
should it be a part of public user docs? 
E.g. sub page in CPU device

Reply:
I would agree, This is more user documentation, rather than developer one.
@wangleis  We already has some section with CPU device doc dedicated multi-threading options https://docs.openvino.ai/2024/openvino-workflow/running-inference/inference-devices-and-modes/cpu-device.html#multi-threading-optimization. So I would propose  to combine this data with the page I shared.

Reply:
@dmitry-gorokhov Since the length of this article is long, is it better to create sub page in CPU device and add link in multi-threading options you shared?

Reply:
@ilya-lavrenov @dmitry-gorokhov Updated